src/instructlab/model/backends/backends.py

```diff
@@ -167,6 +174,13 @@ def is_model_gguf(model_path: pathlib.Path) -> bool:
     # Third Party
     from gguf.constants import GGUF_MAGIC
 
+    if not model_path.is_file():
+        logging.debug(
+            "The path to the model %s is not a file, and therefore cannot be a model in .gguf format.",
```

Comment:
Also the repo tends to do: `logger.debug(f"Message: {var}")` for formatting.

Reply:
Okay no problem, just changed over. In the future we may want to standardize on lazy logging instead- I looked into it a bit and it seems to be a bit faster.